analytics/README.md

```diff
@@ -0,0 +1,26 @@
+# Helsinki Design System Analytics script
+
+A node script to collect the HDS-React package's usage statistics from City of Helsinki Github repositories.
+
+The script does the next steps:
+1. Searches all City of Helsinki TypeScript Github repositories. It filters out the Helsinki Design System repository
```

Comment:
Should we also search for pure JS repositories, or are they so rare that it's not worth it?

Reply:
I did not want to include those because the previous statistics did not